src/openrct2/ride/RideData.h

```diff
@@ -421,7 +464,7 @@ constexpr const RideTypeDescriptor DummyRTD =
     SET_FIELD(CoveredTrackPieces, {}),
     SET_FIELD(StartTrackPiece, TrackElemType::EndStation),
     SET_FIELD(TrackPaintFunction, nullptr),
-    SET_FIELD(Flags, 0),
+    SET_FIELD(Flags, { static_cast<RideTypeFlags>(0) }),
```

Comment:
Won't this set bit 0?

Reply:
I can add an empty constructor but I don't this to be necessary for the dummy rtd.